src/release-notes/sssd-2.10.0.rst

```diff
@@ -0,0 +1,126 @@
+SSSD 2.10.0 Release Notes
+===============================
+
+Highlights
+----------
+
+General information
+~~~~~~~~~~~~~~~~~~~
+
+* **IMPORTANT note for downstream maintainers!**
+
+  This release features significant improvements of "running with less
+  privileges (under   unprivileged service user)" feature. There is still a
+  ``./configure`` option ``--with-sssd-user=`` available that allows downstream
+  package maintainers to choose if support of non-root service user should be
+  built. In case such support is built, a preferred way to configure service
+  user is simply by starting SSSD under this user; for example, using
+  ``User=/Group=`` options of systemd sssd.service file. Upstream defaults are
+  to build ``--with-sssd-user=sssd`` and to install systemd service with
+  ``User=/Group=sssd``. In this case, only several helper processes -
+  ``ldap_child``, ``krb5_child`` and ``selinux_child`` - are executed with
+  elevated capabilities (that are now granted using fine grained file
+  capabilities instead of SUID bit). All other SSSD components run without any
+  capabilities. In this scenario it's still possible to re-configure SSSD to run
+  under ``root`` (if needed for some reason): besides changing ``User/Group=``
+  options, some other tweaks of systemd service files are required.
+
+  A legacy method to configure a service user - sssd.conf ``user`` option - is
+  now deprecated and its support isn’t built by default. It can be enabled using
+  ``--with-conf-service-user-support`` ``./configure`` option if needed (for
+  example, due to backward compatibility requirements of stable releases).
+
+  Further, no matter if SSSD is built ``--with-sssd-user=sssd`` or
+  ``--with-sssd-user=root``, when it's configured to run under ``root`` (in both
+  cases) it still runs without capabilities, the same way as when it's
+  configured to run under ``sssd`` user. The only difference is from the DAC
+  perspective.
+
+  Important note: owner of ``/etc/sssd/sssd.conf`` file (and snippets) should
+  match the user configured to start SSSD service. Upstream spec file changes
+  ownership of existing ``sssd.conf`` to ``sssd`` during package installation
+  for seamless upgrades.
```

Comment:
"Upstream spec file changes ownership of existing ``sssd.conf`` to ``sssd`` during package installation for seamless upgrades."
->
"Upstream spec file and service files change ownership of existing ``sssd.conf`` to ``sssd`` during package installation and at runtime for seamless upgrades / transition period only."